app/views/categories/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="d-flex flex-column justify-content-between align-items-between" style="height: 80vh">
+  <h1 class="mb-3">Categories</h1>
+
+  <div>
+    <% @categories.each do |category| %>
+      <div class="card mb-1">
+        <div class="card-body">
+          <div class="row">
+            <div class="col">
+              <%= image_tag(category.icon, alt: "Icon pic", height: "40px") %>
+            </div>
+            <div class="col">
+              <%= link_to category.name, category_path(category) %>
+            </div>
+            <div class="col">
```

Comment:
Kindly make sure that your application follows the design specifications when it comes to colors, font size, font weight, and spaces between elements as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/211157479-183b2dd2-3bf8-4a0c-819d-40888bb8babe.png)

Your design:

![image](https://user-images.githubusercontent.com/38088818/211157418-a5b01e52-dd54-416b-a215-f46d474e0ed0.png)

Design Specifications:

![image](https://user-images.githubusercontent.com/38088818/211157457-942b19d4-a5f9-4697-afbb-7bcf2495aad7.png)

- Kindly do your best to match the exact design as possible. The font weight of the prices can a bit bolder and more aligned to the right. The font color for the category name and date should be light grey.
- The button text should be uppercase, have a white border and the button should fill up the entire mobile width.